### ISSUE TITLE
.exe logs not generating

### ISSUE THREAD

**Issue by KingVulpes** [NVIDIAGameWorks/rtx-remix#553]
### Describe the bug

For quite awhile now, the .exe logs have not been generating on my Steam Deck, this happens for any game and any runtime

### How do you reproduce the bug?

I'm not sure, I've been told there's not even a way to disable the .exe logs

### What is the expected behavior?

The runtime should be generating the logs

### Version

Any

**Comment by NV-LL** [comment]
Thanks for reaching out. Are you able to get Remix working on any game (i.e., does the Remix banner appear when you launch the game, or does Alt+X work)?

**Comment by KingVulpes** [comment]
> Thanks for reaching out. Are you able to get Remix working on any game (i.e., does the Remix banner appear when you launch the game, or does Alt+X work)?

Yes, Remix still hooks and raytraces, no problems with that, only issue is the .exe log not generating

**Comment by NV-LL** [comment]
REMIX-3339 for tracking
